server/src/main/java/com/broadcom/lsp/cobol/core/strategy/Messages.java

```diff
@@ -1,51 +0,0 @@
-/*
```

Comment:
Why did you remove this class?

Reply:
I guess we don't use this class now and all the messages are being handled through the resource bundle. I will check this again. Did see this yesterday in the Coverity Scan, so thought of removing it.

Reply:
I agree to delete it, but please do it as a separate commit not to bring a mess to this one

Reply:
done